# HG changeset patch
# User Rik <rik@octave.org>
# Date 1544742089 28800
#      Thu Dec 13 15:01:29 2018 -0800
# Node ID b2f806601d306f04f5e522f49f2d272f55a5a17b
# Parent  3af4f3e2dae865bc159a49b577111b9cb53143e3
legend.m: Remove debugging commands from cset 4f044bc43f07.

* legend.m: Remove commented out keyboard command.

diff --git a/scripts/plot/appearance/legend.m b/scripts/plot/appearance/legend.m
--- a/scripts/plot/appearance/legend.m
+++ b/scripts/plot/appearance/legend.m
@@ -876,17 +876,16 @@ function [hleg, hleg_obj, hplot, labels]
 
           switch (typ)
 
             case "line"
               color = get (hplt, "color");
               style = get (hplt, "linestyle");
               lwidth = min (get (hplt, "linewidth"), 5);
               if (! strcmp (style, "none"))
-                #keyboard;
                 l1 = __go_line__ (hlegend, ...
                        "xdata", ([xoffset, xoffset + linelength] + xk * xstep) / lpos(3), ...
                        "ydata", [1, 1] .* (lpos(4) - yoffset - yk * ystep) / lpos(4), ...
                        "color", color, "linestyle", style, ...
                        "linewidth", lwidth, "marker", "none");
                 setappdata (l1, "handle", hplt);
                 hobjects(end+1) = l1;
               endif
